config/FCCAnalysisRun.py

```diff
@@ -596,7 +596,8 @@ def runStages(args, rdfModule, preprocess):
             #run locally
             if runBatch == False:
                 print ('----> Running Locally')
-                runLocal(rdfModule, chunkList[ch], outputchunk, args)
+                args.output = outputchunk
+                runLocal(rdfModule, chunkList[ch], args)
```

Comment:
Good catch. This could have been caught by a test, still need to add some more ...